cmd/testrun_abort.go

```diff
@@ -48,3 +56,20 @@ func testRunAbort(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 }
+
+func testRunAbortAll(cmd *cobra.Command, args []string) {
+	c := NewClient()
+	ok, resp, err := c.TestRunAbortAll()
```

Comment:
I guess we want to at least define the organisation that this command affects, right?

Reply:
Good point, at this point it aborts all runs for all orgs for the user.